webpack.config-frontend.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     arena_contest_contestant:
       './frontend/www/js/omegaup/arena/contest_contestant.ts',
     arena_contest_list: './frontend/www/js/omegaup/arena/contest_list.ts',
+    arena_contest_list2: './frontend/www/js/omegaup/arena/contest_list2.ts',
```

Comment:
```suggestion
    arena_contest_listv2: './frontend/www/js/omegaup/arena/contest_listv2.ts',
```

mismo comentario que en el otro PR con respecto a cambiar el sufijo `2` -> `v2`.